docs/integrations/streamsets.rst

```diff
@@ -0,0 +1,158 @@
+.. _cratedb-streamsets:
+
+================================================================
+Data Stream Pipelines with CrateDB and StreamSets Data Collector
+================================================================
+
+This integration document details how to create data streaming pipelines
+using CrateDB and StreamSets Data Collector.
+
+
+Abstract
+========
+
+The `StreamSets Data Collector`_ is a lightweight and powerful engine that
+allows you to build streaming, batch and change data capture pipelines that
```

Comment:
I don't entirely understand this line - are these all nouns compounded to 'data capture pipelines'? or just to 'pipelines'? Are these three things you can build or is building one of three?

Reply:
Yeah, they all bind to pipelines. There are 3 types:
- Streaming pipelines
- Batch pipelines
- Change-data-capture (CDC) pipelines

What would be a better way of forumulating this?